src/openms/include/OpenMS/ANALYSIS/OPENSWATH/OpenSwathOSWWriter.h

```diff
@@ -270,23 +279,15 @@ namespace OpenMS
                                         << sub_it->getMetaValue("peak_apex_int") << ", " 
                                         << total_mi << "); ";
           }
-          else if (sub_it->metaValueExists("FeatureLevel") && sub_it->getMetaValue("FeatureLevel") == "MS1")
+          else if (sub_it->metaValueExists("FeatureLevel") && sub_it->getMetaValue("FeatureLevel") == "MS1" && sub_it->getIntensity() > 0)
```

Comment:
We only report isotopes if any signal was detected at all. In my benchmark data set, this was the case for ~60% with four isotopes.